feugy/tabulous
# feat(cli): introduces list and deletion commands

### :book: What's in there?

Some love and care for the admin: one CLI command to list all existing players, and two other commands to deletes players and games. The show-player command now displays more details on player's games.

Deletion commands don't ask for confirmation, but they expect exact model ids to operate.

Are included here:

- fix(server): no DB index for players
- fix(cli): does not display owned games properly
- feat(server): allows admins deleting any game
- feat(server): adds player deletion GraphQL mutation
- feat(server): introduces listPlayers GraphQL mutation
- feat(cli): introduces delete-game command
- feat(cli): introduces delete-player command
- feat(cli): introduces list-players command
- feat(cli): displays game kind, creation date and id when showing player details
- feat(cli): displays games ids when showing player details
- chore(server): directly uses repositories from GraphQL resolvers
- chore(cli): ensures new line at the end of the output
- chore(cli): ensures common option docs consistency

### :test_tube: How to test?

This would need to be deployed to be tested.
Locally:
1. run `pnpm migrate` to fix the players DB index
2. run `tabulous list-players` to see all players
3. run `tabulous -u _username_ show-player` to see a player's game and lobby list
4. run `tabulous delete-game _game-id_` to delete a game or lobby
4. run `tabulous delete-player _player-id_` to delete a player account.
